### ISSUE TITLE
Linux: No matching distribution found for python-magic-bin==0.4.14

### ISSUE THREAD

**Issue by aitsvet** [drforse/not-llama-fs#1]
https://stackoverflow.com/a/66732827

[python-magic-bin 0.4.14](https://pypi.org/project/python-magic-bin/0.4.14/#files) provides wheels for OSX, w32 and w64, but not for Linux. And there is no source code at PyPI.

You need to install it from [github](https://github.com/julian-r/python-magic):

```
pip install git+https://github.com/julian-r/python-magic.git
```

**Comment by drforse** [comment]
Hello. I fixed the issue in the latest commit 802854d3e618a8678d16d9a6e85d83b813ff9137 
I checked both on Windows and Linux in clean environments, and it work fine now. Please, check if it works on your computer

**Comment by aitsvet** [comment]
problem solved
